File: C/1045.c
#include <stdio.h>

int main(){

    float a,b,c;
    scanf("%f %f %f",&a,&b,&c);
    float maior, meio, menor;

     if (a >= b && a >= c) {
        maior = a;
        if (b >= c) {
            meio = b;
            menor = c;
        } else {
            meio = c;
            menor = b;
        }
    } else if (b >= a && b >= c) {
        maior = b;
        if (a >= c) {
            meio = a;
            menor = c;
        } else {
            meio = c;
            menor = a;
        }
    } else {
        maior = c;
        if (a >= b) {
            meio = a;
            menor = b;
        } else {
            meio = b;
            menor = a;
        }
    }

    float A = maior;
    float B = meio;
    float C = menor;

    // Verificar tipo de triângulo
    if (A >= B + C) {
        printf("NAO FORMA TRIANGULO\n");
    } else {
        if (A * A == B * B + C * C) {
            printf("TRIANGULO RETANGULO\n");
        }
        if (A * A > B * B + C * C) {
            printf("TRIANGULO OBTUSANGULO\n");
        }
        if (A * A < B * B + C * C) {
            printf("TRIANGULO ACUTANGULO\n");
        }
        if (A == B && B == C) {
            printf("TRIANGULO EQUILATERO\n");
        }
        if ((A == B && A != C) || (A == C && A != B) || (B == C && B != A)) {
            printf("TRIANGULO ISOSCELES\n");
        }
    }

    return 0;
}

File: C/1045.py
x = list(map(float, input().split()))

x.sort()

A = x[2]
B = x[1]
C = x[0]

    # Verificar tipo de triângulo
if A >= B + C:
        print("NAO FORMA TRIANGULO")
else:
    if A * A == B * B + C * C:
            print("TRIANGULO RETANGULO")
    if A * A > B * B + C * C:
            print("TRIANGULO OBTUSANGULO")
    if A * A < B * B + C * C:
            print("TRIANGULO ACUTANGULO")
    if A == B and B == C:
            print("TRIANGULO EQUILATERO")
    if (A == B and A != C) or (A == C and A != B) or (B == C and B != A):
            print("TRIANGULO ISOSCELES")